modules/dhcp/providers/server/native_ms.rb

```diff
@@ -196,6 +230,61 @@ def execute cmd, msg=nil, error_only=false
       response
     end
 
+    # Invokes netsh.exe with a script file
+    def execute_with_script scriptfile, msg=nil, error_only=false
```

Comment:
This method's nearly identical to `execute` just above, the only difference seems to be the command and timeout - could they share the remainder of the code?